department-of-veterans-affairs/va.gov-cms
# VACMS-1798: Fix Outreach Hub Events listing access.

## Description

See #1798. 

## Testing done
Manual

## Screenshots

<img width="1310" alt="Screen Shot 2020-05-20 at 12 10 43 PM" src="https://user-images.githubusercontent.com/16477724/82481732-013f0a00-9a93-11ea-9e96-d6faa6fc235f.png">

**screenshots for engineers to validate access assumptions**
<img width="1092" alt="Screen Shot 2020-05-20 at 11 51 02 AM" src="https://user-images.githubusercontent.com/16477724/82481802-19af2480-9a93-11ea-8831-8dc9c4e4e628.png">
<img width="1192" alt="Screen Shot 2020-05-20 at 11 52 54 AM" src="https://user-images.githubusercontent.com/16477724/82483044-f8e7ce80-9a94-11ea-83a8-fdea39af815e.png">
^ odd that the column storing term id is actually named "section_id"...


## QA steps

**Assumption: users who have access to the following sections should be able to create events in Outreach Hub:**
* `- Veterans Affairs (tid:5 | section_id:7)`
* `-- Office of Public & Intergovernmental Affairs (tid:6 | section_id:9)`
* `--- Outreach Hub (tid:7 | section_id:4)`

Login as admin and assign "Content Editor - VAMC" role to `tucker.patterson@va.gov`

Login as `tucker.patterson@va.gov`. Verify that Tucker is assigned to the Outreach Hub section.
- [x] create an Event - http://pr1800.ci.cms.va.gov/node/add/event and confirm you can select "Outreach and Events: Outreach Events" option in "Event listing" field
- [x] assign Tucker to "Office of Public & Intergovernmental Affairs" section and unassign from "Outreach Hub". Create an Event - http://pr1800.ci.cms.va.gov/node/add/event and confirm you can select "Outreach and Events: Outreach Events" option in "Event listing" field
- [x] assign Tucker to "Veterans Affairs" section and unassign from "Office of Public & Intergovernmental Affairs". Create an Event - http://pr1800.ci.cms.va.gov/node/add/event and confirm you can select "Outreach and Events: Outreach Events" option in "Event listing" field
